docs/spec/exceptions.rst

```diff
@@ -0,0 +1,62 @@
+Exceptions
+==========
+
+Some type checking behaviors, such as type narrowing and reachability analysis,
+require a type checker to understand code flow. Code flow normally proceeds
+from one statement to the next, but some statements such as ``for``, ``while``
+and ``return`` can change the code flow. Similarly, ``try``/``except``/``finally``
+statements affect code flow and therefore can affect type evaluation. For example::
+
+    x = None
+    try:
+        some_function()
+        x = 1
+    except:
+        pass
+
+    # The type of `x` at this point could be None if `some_function` raises
+    # an exception or `Literal[1]` if it doesn't, so a type checker may
+    # choose to narrow its type based on this analysis.
+    reveal_type(x)  # Literal[1] | None
+
+
+Context Managers
+----------------
+
+Context managers may optionally "suppress" exceptions. When such a context
```

Comment:
Maybe briefly remind the reader *how* they do that? (I.e., by returning `True`.)